src/VaultSharp/V1/SecretsEngines/OpenLDAP/LdapCredentials.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace VaultSharp.V1.SecretsEngines.OpenLDAP
+{
+    public class LdapCredentials : UsernamePasswordCredentials
```

Comment:
Recommendation to rename the class to `LDAPCredentials` instead of `LdapCredentials` ensuring that abbreviations are consistent.